receiver.py

```diff
@@ -0,0 +1,78 @@
+"""
+Created on Thu May 27 10:31:35 2021
+
+@author: EPA5COB
+
+"""
+
+import sys
+import json
+from Predefined_values import limit
+    
+def Form_Receiver_input():
+    
+    list_temp =[]
+    list_soc =[]
+    
+    for i in sys.stdin:
+        bms_param = json.loads(i) #creates a dictinary out of 'i'
+        for key, value in bms_param.items():
+            if key in limit.keys():
+                if key == 'Temperature':

```

Comment:
This `if` ladder will keep expanding every time we add a new parameter. All we need to do is segregate into individual lists. You can use a `DataFrame` to do that, or just map each parameter to a list.
One of the nice side-effects of forcing ourselves into lesser complexity - is exploration of language features :)